.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+default_language_version:
+    python: python3.8
```

Comment:
Because of the pre-commit errors (relating to networkx) that we've been seeing with 3.9. We probably don't need to change this, but I've been using 3.8 everywhere else, so I just thought that being consistent would be preferred. 

I don't feel strongly about this though, so this can be reverted.

Reply:
All you had to say was networkx -- I'm on board.